cnf-tests/testsuites/pkg/utils/consts.go

```diff
@@ -114,4 +116,10 @@ const (
 	SriovFecNodeConfigCRDName = "sriovfecnodeconfigs.sriovfec.intel.com"
 	// SriovFecClusterConfigCRDName contains the name of the SriovFecCluster config policies CRD
 	SriovFecClusterConfigCRDName = "sriovfecclusterconfigs.sriovfec.intel.com"
+	// SriovFecClusterConfigName contains the name of the SriovFecCluster config policies allow by the operator
+	// https://github.com/open-ness/openshift-operator/blob/main/sriov-fec/controllers/sriovfecclusterconfig_controller.go#L87
+	SriovFecClusterConfigName = "config"
```

Comment:
If these are local only to the fec test, let's put them in that go file.
Use them here only if they are shared between packages

Reply:
Move the constants :-)